lib/Table/createRegionMappedImageryProvider.ts

```diff
@@ -180,7 +180,7 @@ const getImageryLayerFeatureInfo = action(
       const rowObject = style.tableModel.tableColumns.reduce<{
         [key: string]: string | number | null;
       }>((obj, column) => {
-        obj[column.title] = column.valueFunctionForType(rowId);
+        obj[column.name] = column.valueFunctionForType(rowId);
```

Comment:
Is this to do with mustache templates and renamed table column names?

Reply:
It fixes https://github.com/TerriaJS/terriajs/issues/6228 - as the feature properties are added to mustache context - which come from the `rowObject`.

We used to have "aliases" in the old `FeatureInfoSection.jsx` which meant that both column `title` and `name` could be used in mustache templates.
This was removed at some point, so only column `title` could be used - which caused the bug

With this PR we are getting back to original behavior - where column `name` is used

Reply:
Is it consistent with point csvs?

Reply:
Yep - 

https://github.com/TerriaJS/terriajs/blob/b0d99ac0f0a07b8b06cebcfaa7b59531260ec8ec/lib/Table/createLongitudeLatitudeFeaturePerRow.ts#L89-L97